dvc/repo/__init__.py

```diff
@@ -53,32 +53,120 @@ def wrapper(repo, *args, **kwargs):
 class Repo:
     DVC_DIR = ".dvc"
 
-    from dvc.repo.add import add
-    from dvc.repo.checkout import checkout
-    from dvc.repo.commit import commit
-    from dvc.repo.destroy import destroy
-    from dvc.repo.diff import diff
-    from dvc.repo.fetch import fetch
-    from dvc.repo.freeze import freeze, unfreeze
-    from dvc.repo.gc import gc
-    from dvc.repo.get import get as _get
-    from dvc.repo.get_url import get_url as _get_url
-    from dvc.repo.imp import imp
-    from dvc.repo.imp_url import imp_url
-    from dvc.repo.install import install
-    from dvc.repo.ls import ls as _ls
-    from dvc.repo.move import move
-    from dvc.repo.pull import pull
-    from dvc.repo.push import push
-    from dvc.repo.remove import remove
-    from dvc.repo.reproduce import reproduce
-    from dvc.repo.run import run
-    from dvc.repo.status import status
-    from dvc.repo.update import update
-
-    ls = staticmethod(_ls)
-    get = staticmethod(_get)
-    get_url = staticmethod(_get_url)
+    def add(self, *args, **kwargs):
```

Comment:
What do you think of ignoring the error for now with `# type: ignore[misc]`?

Reply:
Reasonable, because `A full fix seems quite difficult, but this is better than both a crash and a blocking error.` Baned class scoped import is only a temporary solution for the crashes, and we do not meet the crashes for now. We can remove the `# type: ignore[misc]` after it was finally solved.